Index: backports-20200902_001-4.4.60-931c337125/drivers/net/wireless/ath/ath11k/hw.h
===================================================================
--- backports-20200902_001-4.4.60-931c337125.orig/drivers/net/wireless/ath/ath11k/hw.h
+++ backports-20200902_001-4.4.60-931c337125/drivers/net/wireless/ath/ath11k/hw.h
@@ -16,7 +16,7 @@
 /* Max num of stations (per radio) */
 #define TARGET_NUM_STATIONS	128
 #define ATH11K_QMI_TARGET_MEM_MODE	ATH11K_QMI_TARGET_MEM_MODE_512M
-#define ATH11K_DP_TX_COMP_RING_SIZE	8192
+#define ATH11K_DP_TX_COMP_RING_SIZE	4096
 #define ATH11K_DP_RXDMA_MON_STATUS_RING_SIZE	512
 #define ATH11K_DP_RXDMA_MONITOR_BUF_RING_SIZE	128
 #define ATH11K_DP_RXDMA_MONITOR_DST_RING_SIZE	128
